layouts/partials/social.html

```diff
@@ -35,6 +35,16 @@
       <a class="glyph" alt="Keybase profile" href="https://keybase.io/{{ .Site.Params.KeybaseUser }}"><img src="{{ .Site.BaseURL }}icons/keybase.svg" height="24px" width="24px"></a>
     </div>
     {{ end }}
+    {{ with .Site.Params.LinkedInUser }}
+    <div class="icon-24x24">
+      <a class="glyph" alt="LinkedIn profile" href="https://www.linkedin.com/in/{{ . }}"><img src="/icons/linkedin.svg" height="24px" width="24px"></a>
```

Comment:
This is the way I had the social icons set up in the initial release of the theme, with `img src="/icons/icon-name.svg"`. @sauerj pointed out in #27 that if the Hugo site is set up somewhere other than a site's root directory (e.g., at `example.com/blog`), the icon assets won't be reachable, so I've changed it. For consistency, you can have a look at the way other icons are added in this file.